Examples.md

```diff
@@ -319,7 +323,7 @@ spec:
           restartPolicy: Never
           containers:
             - name: run-backup-cron
-              image: clickhouse/clickhouse-client:latest
+              image: clickhouse/clickhouse-server:latest
```

Comment:
old `clickhouse-client` should works with new `clickhouse-server`
which kind of issue you got here?

Reply:
ok. got it
https://hub.docker.com/r/clickhouse/clickhouse-client/tags
doesn't support arm64
let's add comment here